packaging/python_wheel/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM quay.io/pypa/manylinux1_x86_64
+RUN yum install -y zlib-devel bzip2-devel flex
+
+############
+# CMake
+############
+
+# Need to build this from source, since the binaries from Kitware don't
+# run with the old version of the libraries in this Centos version
+RUN wget --no-check-certificate http://cmake.org/files/v3.7/cmake-3.7.2.tar.gz && tar xfz cmake-3.7.2.tar.gz && cd cmake-3.7.2 && ./configure && make -j2 install
+RUN rm -rf cmake-3.7.2.tar.gz cmake-3.7.2
+
+
+############
+# HDF5
+############
+
+RUN wget --no-check-certificate http://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8.17/src/hdf5-1.8.17.tar.bz2 && tar xfj hdf5-1.8.17.tar.bz2
```

Comment:
weird, I can't seem to get this file.  Also, --no-check-certificate isn't needed for http